hashing/te-tag-query-java/com/facebook/threatexchange/TETagQuery.java

```diff
@@ -976,6 +899,11 @@ public void handle(
         } else if (option.equals("--dry-run")) {
           dryRun = true;
 
+        } else if (option.equals("-I")) {
```

Comment:
Maybe I'm missing something, I don't see a definition for the -I in the usage() function.

Reply:
thx